includes/class-danger-zone.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace WeDevs\ERP;
+
+/**
+ * ERP Danger Zone Class
+ * Reset all data of WPErp and make a fresh installation
+ *
+ * @since 1.8.7
+ */
+class DangerZone {
```

Comment:
Kindly use underscore(_) for multi-word class names, `i.e., Danger_Zone` whick is suggested by WPCS

Reply:
Yes, I'm adding. Thank you for your review.